source/tutorials/file-sets.md

```diff
@@ -64,14 +69,14 @@ trace: /home/user (all files in directory)
 :::
 
 Even though file sets conceptually contain local files, these files are *never* added to the Nix store unless explicitly requested.
-You don't have to worry as much about accidentally copying secrets into the world-readable store.
+So You don't have to worry as much about accidentally copying secrets into the world-readable store.
```

Comment:
Capitalisation. Not sure if the connective is really needed here. I would (and did) skip it.

Reply:
Changing to

> Because of that you don't have to worry as much about accidentally copying secrets into the world-readable store.